eth_portal/web3_encoding.py

```diff
@@ -41,3 +49,15 @@ def receipt_content_key(header_hash, chain_id=1):
     # See implementation notes in header_content_key()
     encoded = ssz.encode((chain_id, header_hash), BLOCK_KEY_SEDES)
     return RECEIPT_TYPE_BYTE + encoded
+
+
+def receipt_content_value(receipts) -> bytes:
+    """
+    Compile a list of encoded receipts into their joined content value.
+    """
+    # Awkward quirk that ssz must take an iterable of individual bytes, instead of `bytes`
+    quirky_byte_list = [
+        [bytes([byte]) for byte in receipt.encode()]
```

Comment:
This extra munging can be skipped, after merging this: https://github.com/ethereum/py-ssz/issues/118